src/main/java/org/aksw/simba/squirrel/worker/Worker.java

```diff
@@ -24,8 +24,10 @@
      * @param newUris
      *            the new URIs that have been extracted will be added to this
      *            list.
+     *
+     * @return The timestamp at which the given uri should be recrawled.
      */
-    void performCrawling(CrawleableUri uri, List<CrawleableUri> newUris);
+    public long performCrawling(CrawleableUri uri, List<CrawleableUri> newUris);
```

Comment:
I do not think that the timestamp at which the URI should be recrawled is such a major result of the crawling process that it should be the return type of the method. It should be added to the uri that has been crawled.

But I totally agree to adding the `public` modifier. I do not know why it wasn't there, before... :thinking:

Reply:
@MichaelRoeder I think in interfaces all methods are public implicitly. So it does not matter whether you write public or not. And I removed the return type as you said.